jenkins/Jenkinsfile

```diff
@@ -340,6 +342,15 @@ pipeline {
             }
     }
 
+    stage('Init git submodules') {
+        when { expression { !shouldAbortBuild } }
+        steps {
+            sh """
+                git submodule update --init --recursive
```

Comment:
This is the same as `git submodule init && git submodule update`. I couldn't find a way to insert it before the forking happens within the next stage so had to create a new stage  😞

Reply:
I think this is fine, no problem with more separation of code in this massive job